Sources/Logger/Loggers/FileLogger/FileLogger.swift

```diff
@@ -11,15 +11,22 @@ import Zip
 enum FileLoggerError: Error {
     case missingWritableFileHandle
     case stringToDataConversionFailure
+    case userDefaultsInitFailure
+}
+
+public enum SharingConfiguration {
+    case shared(appGroupID: String)
+    case nonShared(suiteName: String)
+    case defaultNonShared
 }
 
 public class FileLogger: Logging {
     // MARK: - Stored properties
 
     private let logFilePathExtension: String = "log"
-    private let fileManager: FileManager
-    private let userDefaults: UserDefaults
-    private let suiteName: String?
+    let fileManager: FileManager
+    let userDefaults: UserDefaults
```

Comment:
Nestaci ti `private`?

Reply:
nestaci

Reply:
A prečo @jiledaa? Je to kvôli testom? Ak áno tak to mi príde trochu blbé, že sa zmeniť property access level len kvôli testom. Možno radšej pridať nejaké gettery na veci, ktoré v testoch potrebujeme?